content/07.pathogenesis.md

```diff
@@ -134,3 +134,41 @@ The network analysis showed in particular that non-structural proteins 9 and 10
+However, in contrast to high-MOI, low-MOI SARS-CoV-2-infected A549-ACE2 cells showed no significant IFN-I or IFN-III expression, suggesting that IFN induction is MOI dependent. 
+Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III. 
+Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response. 
+However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 
+This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 _ORF3b_ gene, which is considerably longer.
+
+A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line). 
+They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV-1 at different hpi (4-36 hours) with an MOI of 0.3. 
+Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads). 
+Based on the visual inspection (microscopy images) and transcriptional profiling, authors showed distinct host responses after the infection. 
+In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24hpi, and a moderate IFN induction with a strong up-regulation of ISGs. 
+Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells (in contrast to A549-ACE2 cells). 
+The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935]. 
+We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available result files from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
+In general, we found high correlations among the same host (e.g. Calu-3 cells) irrespective of the study and the virus infected, suggesting that the response is host-specific. 
```

Comment:
Please define "high correlations" in terms of a statistical measurement (such as Pearson correlation coefficient > XX)